### ISSUE TITLE
[Accessibility] Add meter ARIA role

### ISSUE THREAD

**Issue by Rumyra** [mdn/content#8610]
## Acceptance Criteria

- [ ] The listed features are documented sufficiently on MDN
- [ ] BCD is updated
- [ ] Interactive example and data repos are updated if appropriate
- [ ] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [ ] Set bugs to `dev-doc-complete`
- [ ] Add entry to Firefox release notes if feature is enabled in release
or
- [ ] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Features to document

The ARIA role `meter` has been added to FF and should be added to the roles page on MDN

## Related Gecko bugs

https://bugzilla.mozilla.org/show_bug.cgi?id=1727616

**Comment by Rumyra** [comment]
Hey @rachelandrew - I'm going to take this off your hands 👍  - It looks like it's kinda done tho 😎

**Comment by rachelandrew** [comment]
ha - I literally just opened it to check (I'm tidying up).
